src/why3prover.ml

```diff
@@ -0,0 +1,74 @@
+(** Calling provers in Why3 *)
+
+open Why3
+
+
+(* list of provers' name *)
+let provers_name : (string * string) list ref =
+    ref
+    [
+        "altergo", "Alt-Ergo";
+        "eprover", "Eprover";
+    ]
+
+(* get the real name of a prover *)
+let get_name : string -> string = fun s ->
+    try
+        List.assoc s !provers_name
+    with
+        Not_found  -> failwith ("Failed to find `" ^ s ^ "`")
+
+(* get why3 config *)
+let config : Whyconf.config = Whyconf.read_config None
+
+(* get the main config *)
+let main : Whyconf.main = Whyconf.get_main config
+
+(* get a prover from the config file *)
```

Comment:
What is this config file? Could you give more details, and links to Why3 documentation?

Reply:
Added in 6563330a83172fc899050b8af25fb14c59c49f28